tests/test_api.py

```diff
@@ -655,9 +683,52 @@ def test_visualize_tangled_tree_layers(self, client, figure_type, data_model_jso
         assert response.status_code == 200
 
 
+@pytest.mark.schematic_api
+class TestValidationBenchmark():
```

Comment:
Could you add more comments related to the rationale of the test? From what I understand, you started by getting all of the mock component attributes and isolated them in a manifest. Then, you increased the part that you isolated to 1000 rows and calculate run time of that manifest?

Reply:
Sure, added